src/components/forms/ReviewNHSAppTemplate/ReviewNHSAppTemplate.tsx

```diff
@@ -23,9 +23,9 @@ export function ReviewNHSAppTemplate({
     router.push(state.redirect);
   }
 
-  const { nhsAppTemplateName, nhsAppTemplateMessage } = state;
+  const { name, message } = state.NHS_APP!;
```

Comment:
What would happen is NHS_APP is undefined. Is this likely to happen?

Reply:
messed around with the types a bit, the typecast wasn't needed and it's been removed now